packages/riipen-ui/src/components/Input.jsx

```diff
@@ -47,6 +48,7 @@ const Input = ({
   const componentClassName = clsx(
     error ? "error" : null,
     disabled ? "disabled" : null,
+    Icon ? "iconInput" : null,
```

Comment:
Not stoked on this class name but couldn't think of a better idea to get that left padding

Reply:
Maybe just `iconPadding`? :shrug: